build.gradle

```diff
@@ -31,7 +31,7 @@ ospackage {
 
     into '/usr/share/elasticsearch/plugins'
     from(zipTree(project.file(archivePath).absolutePath)) {
-        into "opendistro_security"
+        into "opendistro-security"
```

Comment:
Is it necessary to make changes on line 56 as well?

Reply:
Renamed on line 56. And resued the `packageName` defined in the script.